src/public/services/PublicFormAuthService.ts

```diff
@@ -31,3 +33,11 @@ export const validateEsrvcId = (
     )
     .then(({ data }) => data)
 }
+
+export const spcpLogout = (
```

Comment:
nit: we usually try to name functions with verbs, e.g. `logoutOfSpcpSession`

Reply:
thanks! edited